PopcornEditor/src/butter.js

```diff
@@ -922,67 +922,127 @@ window.Butter = {
           finishedCallback( project );
         }
 
+        function initialMediaSources( qs ){
```

Comment:
Thinking how to generalize this bit... multiple URLs is relatively straightforward, but customizing short-form expansion probably warrants a callback into the embedding host.

This could be done through events/event.js by sending an event up to the parent frame (or local embedding code, if we retool event.js to support that too) asking for initial project data and having the host send a reply back.

That might be more time-efficient than waiting for the full loaded event. Not sure what implications there are... @nolski, thoughts?